_posts/2020-07-14-mill-brook-summer-2020.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: post
+title: Mill Brook Summer 2020
+description: As we all try and adjust to the changes each day brings, the beauty of nature always awaits us outside. Here are some photos of our community gardens, wildlife, and the Path’s calming vistas. For everyone’s safety and protection, please wear a mask while out walking. Thank you.
```

Comment:
for each post, the first paragraph shows up in the list page. since now the first paragraph are all the images, they will all show up on the list page. instead simply move the description as the first paragraph of the post content.

<img width="584" alt="image" src="https://user-images.githubusercontent.com/932940/87500961-ea004100-c62b-11ea-9bad-1f4686ba18b8.png">